src/main/java/com/app/katchup/DistributedCalendar/CalendarService.java

```diff
@@ -1,33 +1,24 @@
 package com.app.katchup.DistributedCalendar;
 
-
 import com.app.katchup.MeetingResponse.MeetingResponseRepository;
 import com.app.katchup.MeetingResponse.model.Decision;
-import com.app.katchup.Users.User;
+import com.app.katchup.MeetingResponse.model.MeetingID;
 import com.app.katchup.Users.UserRepository;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 
 @Service
 public class CalendarService {
-
-    private static final Logger logger = LogManager.getLogger(com.app.katchup.Users.UserService.class);
     UserRepository userRepository;
+    private static final Logger logger = LogManager.getLogger(com.app.katchup.Users.UserService.class);
```

Comment:
Logger class error. have CalendarService.class

Reply:
changed it to DistributedCalendar.CalendarService.class

Reply:
removing it infact as I am not using logger in this service class for now.